app/models/rental.py

```diff
@@ -0,0 +1,82 @@
+
+from sqlalchemy.orm import relationship
+from app.models.video import Video
+from app.models.customer import Customer
+from flask import current_app
+from app import db
+from datetime import timedelta, datetime
+# import datetime
+from flask import make_response
+
+
+class Rental(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    customer_id = db.Column(db.Integer, db.ForeignKey('customer.customer_id'))
+    video_id = db.Column(db.Integer, db.ForeignKey('video.video_id'))
+    due_date = db.Column(db.DateTime)
+
+    video = relationship("Video", back_populates="rentals")
+    customer = relationship("Customer", back_populates="rentals")
+    
+    
+    @classmethod
+    def check_out(cls, customer_id, video_id):
+
+        customer = Customer.query.get(customer_id)
+        video = Video.query.get(video_id)
+
+        rental= cls(customer_id = customer_id,
+                            video_id = video_id,
+                            due_date = datetime.today() + timedelta(days=7))
+        
+        if video.available_inventory < 0:
+            return make_response({"details": "Video out of stock"}, 400)
```

Comment:
I suggest either  raising an error, or returning some kind of error object and letting the `routes.py` file handle the specific json and request-response code.